### ISSUE TITLE
No config files

### ISSUE THREAD

**Issue by highlabs** [railsware/bozon#9]
Hi! :)

I'm installer Bozon 0.3.4 via `npm install bozon -g`. But when I run `bozon new appname`, bozon generate a directory withou `gulpfile.js` and `package.json`. :/ I cannot run `bozon start`, because: 
```
➜  appname bozon start
[11:08:42] No gulpfile found
```
I tried to run `npm install` too... :/

My system:
```
➜  appname node --version
v5.9.0
➜  appname npm --version
3.7.3
➜  appname sw_vers
ProductName:	Mac OS X
ProductVersion:	10.11
BuildVersion:	15A284

```

**Comment by alchapone** [comment]
@highlabs  This one sounds really serious.. But I can not reproduce it with v0.3.4. Could you provide an output of `bozon new appname` ?

**Comment by highlabs** [comment]
I tried this and worked: http://pastebin.com/8xc5s1su

But, some times the script not init the `npm install`: http://pastebin.com/zpMXkZbp

I'm trying to get what happening. xD

**Comment by alchapone** [comment]
Ok, cool so now we've got that bug: If you enter and app name with underscore in it, generator silently fails at some point. I'll take a look at this issue.
P.S. meanwhile it is solved you can build apps which name doesn't contain `_` ;)

**Comment by jiuyuehe** [comment]
you can publish v0.3.5 @alchapone

**Comment by alchapone** [comment]
Addressed in [v0.3.5](https://github.com/railsware/bozon/releases/tag/v0.3.5)
